sklearn/multiclass.py

```diff
@@ -400,8 +406,7 @@ def _fit_ovo_binary(estimator, X, y, i, j):
     y_binary[y == i] = 0
     y_binary[y == j] = 1
     ind = np.arange(X.shape[0])
-    return _fit_binary(estimator, X[ind[cond]], y_binary, classes=[i, j])
-
+    return _fit_binary(estimator, _safe_split(estimator, X, y, indices=ind[cond])[0], y_binary, classes=[i, j])
```

Comment:
sorry, my mistake. you shouldn't be passing y into `_safe_split`. Rather, `_safe_split` should probably allow `y=None`.